### PR TITLE
Sync with jshint master

### PR DESCRIPTION
**Pull request by takenspc**
browser
- Comment, Document, DocumentFragment, Event, Range, Text are defined by [DOM Standard](https://dom.spec.whatwg.org/)
- Notification is defined by [Notification API Standard](https://notifications.spec.whatwg.org/)

worker
- FileReaderSync is defined by [File API](http://www.w3.org/TR/FileAPI/#readingOnThreads)

node
- Several predefined global objects are writable so that they should have "true"

jasmine
- [jasmine 1.3](http://jasmine.github.io/1.3/introduction.html#section-Asynchronous_Support) has runs method
- [jasmine-jquery](https://github.com/velesin/jasmine-jquery) provides loadFixtures and setFixtures

rhino
- [A MDN document](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/Rhino/Scripting_Java) covers both importClass and Packages

### CODE REVIEW

**Comment by takenspc** [comment]
I'm planning to write patch that makes jshint use globals. So, I will be happy if jshint and globals provide same globals.

File: globals.json [hunk 1]
```diff
@@ -437,27 +443,28 @@
...
-		"self": true
+		"self": true,
+		"FileReaderSync": true
 	},
 	"node": {
 		"__filename": false,
 		"__dirname": false,
 		"arguments": false,
-		"Buffer": false,
+		"Buffer": true,
```
Comment (nzakas) [inline globals.json]: Why are these true?

Comment (takenspc) [inline globals.json]: According to the original issue, developers overwrite Buffer, console and so on in some cases.
https://github.com/jshint/jshint/pull/1426

I think that it is not good thing. So I will remove the changes for "node" environment.

File: globals.json [hunk 2]
```diff
@@ -496,6 +503,9 @@
 		"it": false,
 		"jasmine": false,
 		"pending": false,
+		"loadFixtures": false,
```
Comment (takenspc) [inline globals.json]: jasmine itself doesn't have loadFixtures and setFixtures. So I will remove them from this PR.

**Comment by nzakas** [comment]
:+1:

**Comment by es128** [comment]
:+1:

**Comment by sindresorhus** [comment]
Thanks @takenspc :) Released as 6.0.0. Let us know how the JSHint PR goes.

**Pull request merged**